From 8248067e84198797d3f19313c27316cd5bbd083e Mon Sep 17 00:00:00 2001
From: Simon Dubray <simonx.dubray@intel.com>
Date: Wed, 8 Jul 2015 11:53:27 +0200
Subject: [PATCH] Reduce mmc autosuspend delay

Long autosuspend delay (3s) for mmc is reducing the s0ix residency.
This patch decreases it to have a more power friendly behaviour.

Change-Id: Iaf5cd0e93f759198a8c1f92c3986eb5796f29c66
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-13347
Signed-off-by: Simon Dubray <simonx.dubray@intel.com>
---
 drivers/mmc/card/block.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/card/block.c b/drivers/mmc/card/block.c
index e9305786a915..8b541bac9731 100644
--- a/drivers/mmc/card/block.c
+++ b/drivers/mmc/card/block.c
@@ -2959,7 +2959,7 @@ static int mmc_blk_probe(struct mmc_card *card)
 	/* use async mode for suspend/resume */
 	device_enable_async_suspend(&card->dev);
 
-	pm_runtime_set_autosuspend_delay(&card->dev, 3000);
+	pm_runtime_set_autosuspend_delay(&card->dev, 100);
 	pm_runtime_use_autosuspend(&card->dev);
 
 	/*
-- 
1.9.1

